src/hydrate/values/readFundingIfExists.ts

```diff
@@ -1,4 +1,4 @@
-import fs from "node:fs/promises";
+import { promises as fs } from "node:fs";
```

Comment:
[Question] Any particular reason for this? I've been moving towards `node:fs/promises` just to not have to use the `as`, but that's not a hard preference on my end.

Reply:
@JoshuaKGoldberg The reason was that any other attempt to mock fs/promises has failed for me, until this attempt succeeded.
It seems to be a common issue when trying to mock "fs" with testing libraries

Reply:
Got it - I pushed up 9520ed8 just to show an example. Time to merge! 🚀

Reply:
That was convenient!